batchflow/named_expr.py

```diff
@@ -519,7 +519,7 @@ def __init__(self, name=None, mode='w', **kwargs):
     def get(self, **kwargs):
         """ Return a value of a pipeline config """
         name, pipeline, _ = self._get_params(**kwargs)
-        config = pipeline.config or {}
+        config = pipeline.config or Config()
```

Comment:
That would still fail to work if the pipeline config is a dictionary
It is better to remove the keyword in `get` call

Reply:
Provided config is always converted to `Config` (see `Pipeline.__init__`, line `92`), so dict as a config will ruin nothing.